scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/mkcloud.config.j2

```diff
@@ -67,7 +67,9 @@ export scenario={{ admin_scripts_path }}/scenarios/{{ scenario_file }}
 # export cinder_netapp_login=
 # export cinder_netapp_password=
 
-# export UPDATEREPOS=
```

Comment:
We should set up UPDATEREPOS_EXTRA instead of UPDATEREPOS, qa_crowbarsetup should take care of it correctly nevertheless. It can be done later though, once we have UPDATEREPOS from updates_list working.

Reply:
I didn't see any difference between the two and UPDATEREPOS is what's being used by the openstack-mkcloud job, so I did the same, to preserve functionality.